.github/ISSUE_TEMPLATE/20-feature_request.yaml

```diff
@@ -0,0 +1,65 @@
+name: Feature
+description: |
+    Suggest a new feature for next Bareos
+    (**NOTE**: we only making critical & bug fixes to Bareos current)
+title: Bareos Next - Feature request
+labels: feature request, bareos next
```

Comment:
the default GitHub label for this is "enhancement", we should probably re-use that one.